### PR TITLE
Flask5 homework

### PR DESCRIPTION
**Pull request by oleksii-manzik**

### CODE REVIEW

File: flask5/hotel/fields_check.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from flask_restful import abort
```
Comment (dimigor) [inline flask5/hotel/fields_check.py]: name file like `..._util.py`

File: flask5/hotel/fields_check.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+
+def check_if_all_parameters_is_filled(args):
+    if not all(args.values()):
+        abort(400, error_code=400,
```
Comment (dimigor) [inline flask5/hotel/fields_check.py]: do not use here flask functions, just return message. Where you call these functions you check if message then abort

File: flask5/hotel/resources.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    def post(self):
+        args = rooms_parser.parse_args(strict=True)
+        check_if_all_parameters_is_filled(args)
+        try:
+            db.session.add(RoomsModel(**args))
+            db.session.commit()
+        except IntegrityError:
+            abort(409, error_code=409,
+                  message=f'Room {args["number"]} is already exists!')
+        return f'New room {args["number"]} was successfully added'
```
Comment (dimigor) [inline flask5/hotel/resources.py]: return all room fields.
client should know what room with what fields was created

File: flask5/hotel/resources.py [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+    def put(self):
+        args = rooms_parser.parse_args(strict=True)
+        check_if_key_in_args('number', args, 'room')
+        check_if_all_parameters_is_filled(args)
+        room = RoomsModel.query.filter_by(number=args['number'])
+        check_if_row_was_affected(room.first(), 'Room', args["number"])
+        room.update(args)
+        db.session.commit()
+        return f'Room {args["number"]} was successfully updated'
```
Comment (dimigor) [inline flask5/hotel/resources.py]: as above

File: flask_lesson_2/app.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+@app.route('/request', methods=['POST'])
+def request_object_example():
+    data = {
+        "request.method": request.method,
+        "request.args": request.args,
+        "request.headers": request.headers,
+        "request.data": request.data
+    }
+    print(data)
```
Comment (dimigor) [inline flask_lesson_2/app.py]: delete print

**Review by dimigor — CHANGES_REQUESTED** [review]

File: flask_lesson_2/app.py (hunk 5 above)
Comment (oleksii-manzik) [reply flask_lesson_2/app.py]: it is not my print. hehe :)

File: flask4/hotel/db_utils.py [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+            f'SELECT * FROM {left_letters_and_numbers_only(table)};'
+        )
+        return cursor.fetchall()
+
+
+def get_with_filter(table: str, column: str, values: list):
+    with sqlite3.connect('hotel_db.db') as con:
+        cursor = con.cursor()
+        cursor.execute(
+            f'''SELECT * FROM {left_letters_and_numbers_only(table)}
```
Comment (dimigor) [inline flask4/hotel/db_utils.py]: wow, what is ^M?

File: flask4/hotel/fields_check.py [hunk 7]
```diff
@@ -0,0 +1,26 @@
+from flask_restful import abort
+
+
+def check_if_all_parameters_is_filled(args):
+    if not all(args.values()):
+        abort(400, error_code=400,
```
Comment (dimigor) [inline flask4/hotel/fields_check.py]: do not abort here, return only message, abort shoud be in views

**Review by dimigor — DISMISSED** [review]

**Pull request merged**